cmd/terraform.go

```
@@ -64,9 +64,32 @@
+	}
+
+	skip, err := flags.GetStringSlice("skip")
+	if err != nil {
+		u.PrintErrorMarkdownAndExit("", err, "")
+	}
+
+	info.ProcessTemplates = processTemplates
+	info.ProcessFunctions = processYamlFunctions
+	info.Skip = skip
+
+	err = e.ExecuteTerraform(info)
```

golangci-lint (1.64.8)

[error] ineffectual assignment to err

(ineffassign)